### ISSUE TITLE
Export ConfigOption

### ISSUE THREAD

**Issue by Knoxvillekm** [formly-js/ng-formly#375]
**I'm submitting a bug report**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```
I am write my own components for ng-formly and I want import ConfigOption from 'ng-formly' for set it in FormlyModule.forRoot({...}) but this is not work.

**Comment by aitboudad** [comment]
@Knoxvillekm can you provide a PR ?

**Comment by Knoxvillekm** [comment]
Just add ConfigOption to src/core/core.ts 27 line

**Comment by Knoxvillekm** [comment]
And how canI fix lint error "The property "[valid, formControl, field, to]" that you're trying to access does not exist in the class declaration." in components html.  Components extends FieldType and have this properties.

**Comment by aitboudad** [comment]
@Knoxvillekm can you open another issue and provide us more details, thanks

**Comment by aitboudad** [comment]
fixed by #378
